ProductionSystem/scripts/dirac-prod-get-trans.py

```diff
@@ -39,14 +39,27 @@
 
 if res['OK']:
   transList = res['Value']
-  for trans in transList:
-    transIDs.append(trans['TransformationID'])
+  if len(transList) == 0:
+    DIRAC.gLogger.notice('No transformation associated with production %s' % prodID)
```

Comment:
Should this be an `error`?

Reply:
Not necessarly. One could create a production object with no transformations associated and then use add transformations using dirac-prod-add-trans script.